README.md

```diff
@@ -419,197 +745,512 @@ const myToolbar = {
-array of `Config.Tools` string constants, optional
+```js
+<DocumentView
+  annotationMenuItems={[Config.AnnotationMenu.search, Config.AnnotationMenu.share]}
+/>
+```
+##### overrideAnnotationMenuBehavior
+array of string, optional, default to none
 
-Defines annotation types that will not show the default annotation menu
-##### annotationMenuItems
-array of `Config.AnnotationMenu` string constants, optional
+Defines menu items that should skip default behavior. Strings should be [Config.AnnotationMenu](./src/Config/Config.js) constants.
 
-Defines menu items that can show when an annotation is selected.
-##### overrideAnnotationMenuBehavior
-array of `Config.AnnotationMenu` string constants, optional
+```js
+<DocumentView
+  overrideAnnotationMenuBehavior={[Config.AnnotationMenu.copy]}
+/>
+```
 
-Defines menu items that should skip default behavior.
 ##### onAnnotationMenuPress
 function, optional
 
-Defines what happens on annotation menu press if it is passed in to `overrideAnnotationMenuBehavior`
+This function is called if the pressed annotation menu item is passed in to [`overrideAnnotationMenuBehavior`](#overrideAnnotationMenuBehavior)
 
 Parameters:
 
 Name | Type | Description
 --- | --- | ---
-annotationMenu | string | One of `Config.AnnotationMenu` string constants
+annotationMenu | string | One of [Config.AnnotationMenu](./src/Config/Config.js) constants, representing which item has been pressed
 annotations | array | An array of `{id, rect}` objects, where `id` is the annotation identifier and `rect={x1, y1, x2, y2}` specifies the annotation's screen rect.
 
+```js
+<DocumentView
+  onAnnotationMenuPress = {({annotationMenu, annotations}) => {
+    console.log('Annotation menu item', annotationMenu, 'has been pressed');
+    annotations.forEach(annotation => {
+      console.log('The id of selected annotation is', annotation.id);
+      console.log('The lower left corner of selected annotation is', annotation.x1, annotation.y1);
+    });
+  }}
+/>
+```
+
 ##### longPressMenuEnabled
 bool, optional, default to true
 
-If true, the viewer will show the default menu on long press.
+Defines whether to show menu of options after long press on text or blank space in the viewer
+
+```js
+<DocumentView
+  longPressMenuEnabled={true}
+/>
+```
 
 ##### longPressMenuItems
-array of `Config.LongPressMenu` string constants, optional
+array of string, optional, default to containing all the items
+
+Defines menu items that can show when long press on text or blank space. Strings should be [Config.LongPressMenu](./src/Config/Config.js) constants.
+
+```js
+<DocumentView
+  longPressMenuItems={[Config.LongPressMenu.copy, Config.LongPressMenu.read]}
+/>
+```
 
-Defines menu items that can show when long press on text or blank space.
 ##### overrideLongPressMenuBehavior
-array of `Config.LongPressMenu` string constants, optional
+array of strings, optional, default to none
+
+Defines menu items on long press that should skip default behavior.
```

Comment:
So, does this mean that the menu item is still displayed, but it gets a callback when pressed? Which callback? That should be mentioned here.